OutOfSchool/OutOfSchool.WebApi.Tests/Services/StatisticServiceTest.cs

```diff
@@ -16,6 +16,7 @@
 using OutOfSchool.Services.Repository;
 using OutOfSchool.Tests.Common.TestDataGenerators;
 using OutOfSchool.WebApi.Models;
+using OutOfSchool.WebApi.Models.Workshop;
```

Comment:
(Redundant code) This `using` might be moved out

Reply:
And now I see that you moved it to other namespace. Previous comment is invalid, closing it